rx-preferences-sample/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
   }
 
   packagingOptions {
-    exclude 'META-INF/rxjava.properties' // rxjava-2.0.0-RC3.jar & rxjava-1.2.0.jar
+    exclude 'META-INF/rxjava.properties' // rxjava-2.0.1.jar & rxjava-1.2.0.jar
```

Comment:
This should probably be part of https://github.com/f2prateek/rx-preferences/pull/56/files